Library/do/bar-better-ci.do

```diff
@@ -17,12 +17,15 @@
 
     betterbar                     ///
         (dr_3 correct_treatment)  ///
-        (med_b2_any_antibiotic med_b2_any_steroid med_b2_any_antister med_l_any_2     ///
-        med_b2_any_schedule_h med_b2_any_schedule_h1 med_b2_any_schedule_x med_l_any_1),  ///
-        over(city) xlab($pct) se bin  ///
-        legend(pos(5) ring(0) c(1) symxsize(small) symysize(small)) ///
-        ysize(7) n barlab(upper)      ///
-        title ("Horizontal Cluster Bar Plot" , justification(left) color(black) span pos(17))
+        (med_b2_any_antibiotic med_b2_any_steroid med_b2_any_antister ///
+		    med_l_any_2 med_b2_any_schedule_h med_b2_any_schedule_h1  ///
+			med_b2_any_schedule_x med_l_any_1)                        ///
+	   ,                                                              /// 
```

Comment:
@luizaandrade 
"," the place of comma is a little bit confusing, what would be the best practice?